bin/node/src/commands/mod.rs

```diff
@@ -24,3 +26,77 @@ const DEFAULT_NTX_TICKER_INTERVAL: Duration = Duration::from_millis(200);
 fn duration_to_human_readable_string(duration: Duration) -> String {
     humantime::format_duration(duration).to_string()
 }
+
+/// Configuration for the Network Transaction Builder component
+#[derive(clap::Args)]
+pub struct NtxBuilderConfig {
+    /// Disable spawning the network transaction builder.
+    #[arg(long = "no-ntb", default_value_t = false)]
+    pub disabled: bool,
```

Comment:
Should we define this as conflicting with `tx_prover_url` and `ticker_interval`? Basically, if we set `disabled`, we shouldn't set the other two.

Reply:
Those are not conflicting meaning that will not panic/crash , but wouldn't be used if disabled is set. I can change it to enforce, wdyt?

Reply:
Yeah, maybe worth enforcing this to make it clear to the user that some combinations are meaningless. At least I don't see any real downsides to doing this.